CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
 - The `string` module gains the `first`, `last`, and `capitalise` functions.
 - Fixed a bug where `string.reverse` would break utf8 strings on target JavaScript.
 - Fixed a bug where `string.slice` would break utf8 strings on target JavaScript.
+- The grapheme iterator used by `string.graphemes` is now locale independent on target JavaScript.
+- Added more utf8 grapheme related tests.
```

Comment:
Could you remove this line? It's not a change from the user's point of view so no need to share this information